t/strict-booleans.t

```diff
@@ -13,11 +14,7 @@ use Workflow::Persister::File;
 
 $Workflow::Condition::STRICT_BOOLEANS = 0;
 
-
-my $persist_dir = catdir( rel2abs( curdir() ), 'tmp_file' );
-unless ( -d $persist_dir ) {
-    mkdir( $persist_dir, 0777 );
-}
+my $persist_dir = tempdir( 'tmp_test_XXXX', DIR => curdir(), CLEANUP => 1 );
```

Comment:
As above.

Should the directory name be a timestamp with high resolution or UUID or similar?